src/api/glue-api.js

```diff
@@ -3,13 +3,12 @@ import { BASE_URL } from '../settings/constants'
 
 const resultPage = '/listings?categoryPage=RESULT'
 const listingPage = '/page'
-const villageHouse = '&unitSubTypes=VILLAGE_HOUSE&unitTypes=HOME&usageTypes=RESIDENTIAL&unitTypesV3=VILLAGE_HOUSE'
+const villageHouse = '&unitSubTypes=VILLAGE_HOUSE&unitTypes=HOME&usageTypes=RESIDENTIAL&unitTypesV3=VILLAGE_HOUSE&text=Casa+de+Vila'
```

Comment:
You could create a function to mount theses query params, so you not necessary crate one by one. What do you think?

Reply:
Ive created a function, but I still have to pass the params for now.